tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_tks.yml

```diff
@@ -9,6 +9,13 @@
 
 - meta: flush_handlers
 
+- name: Set debug log level to 5
+  lineinfile:
+    path: /var/lib/pki/{{variable.TKS_INSTANCE_NAME}}/conf/tks/CS.cfg
```

Comment:
Minor: same comment as before

Reply:
Currently i have excluded the playbook changes for another MR. So marking this comment resolved for this MR.
This MR consists changes related to executing existing tests in Fedora32.